src/lib/slacko.ml

```diff
@@ -18,6 +18,7 @@
 * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 *)
 
+open Internal
```

Comment:
Could you please avoid `open`? I find it very hard for navigating code when functions come out of seemingly nowhere. I prefer either fully qualifified identifiers or module aliases.